static/src/javascripts/projects/common/views/experiments/quick-survey.html

```diff
@@ -0,0 +1,92 @@
+<style>
```

Comment:
No problem! :) For an example - there is an experiments folder for stylesheets (same as JS) here - https://github.com/guardian/frontend/tree/master/static/src/stylesheets/module/experiments - if you put your styles in there, you can then just import them into [common](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/_common.scss)